packages/domains-table/src/domains-table/__tests__/index.tsx

```diff
@@ -1,7 +1,7 @@
 /**
  * @jest-environment jsdom
  */
-import { screen, waitFor, within } from '@testing-library/react';
+import { fireEvent, screen, waitFor, within } from '@testing-library/react';
```

Comment:
fwiw it's better to user the `@testing-library/user-event` library where possible instead of firing raw events. In this case it doesn't make much difference, it just means it more realistically mocks the user action. Like sending mouse overs, mouse downs, mouse ups, clicks events, etc. all in the correct order.
https://testing-library.com/docs/user-event/intro/